### DISCUSSION TITLE
Keep icons in the centre of the screen

### DISCUSSION THREAD

**Discussion by MikeJohnPage** [rxhanson/Charmstone-Community#47] [Ideas]
When using the arrow keys to select an app, the grid of icons appear where the cursor was last placed on the screen. Sometimes when I am accessing an app in one of the far locations, this results in me having to scan the screen to find the grid of icons to identify the correct one.

Ideally, the grid of icons would always remain centred in the screen when using the arrow keys. From my perspective, this would create a consistent interface and a better user experience for those who ingrequently use the mouse and instead are heavy keyboard users.

**Comment by rxhanson** [comment]
Good point, makes sense to me. I'll add this to my backlog.
